server_action_logging/models/ir_actions_server.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2024 Camptocamp (https://www.camptocamp.com).
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+
+import logging
+
+from odoo import fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class IrActionsServer(models.Model):
+    _inherit = "ir.actions.server"
+
+    enable_sql_debug = fields.Boolean(
+        string="Enable SQL Debug",
+        help="Enable SQL Debug for this action",
+        default=True,
```

Comment:
Do you really want to set `True` as default value? That means it will get enabled for all the existing actions...

If that is the case, I'd suggest to adapt the readme because "if action's field ``Enable SQL Debug`` is set, queries are logged too during the action's execution" makes me think it has to be set manually by the user.
IMO it would be more telling to write something like "SQL queries will be logged during the action's execution, what could be deactivated through the ``Enable SQL Debug`` field".

Reply:
Mh, I inherited the code from #2928 and really didn't think about the field declaration. It would make sense to have the field set to `False` instead to avoid too much log pollution. I'll update it asap.